tests/default.toml

```diff
@@ -1,8 +1,9 @@
+#:schema ./match.json
```

Comment:
Some of the match tomls are not using an url?

Reply:
good catch!